components/AppBar/src/MDCAppBarViewController.h

```diff
@@ -51,6 +52,31 @@
  */
 @property(nonatomic) CGFloat headerStackViewOffset;
 
+/** The safe area delegate can be queried for the correct way to calculate safe areas. */
```

Comment:
What is the safe area delegate? It's not immediately what this comment has to do with the property declaration below it.

Reply:
+1 The comment doesn't seem to be relevant to this delegate?

Reply:
Oops, thank you for catching. Updated.